setup-issue-templates/draft-schedule.md

```diff
@@ -1,4 +1,4 @@
 Update `workshop/SCHEDULE.md` to point to appropriate materials for your workshop.
 
   - Add relative links to PDFs in `slides` to the schedule.
-  - We recommend using [htmlpreview](https://github.com/htmlpreview/htmlpreview.github.com) to display rendered versions of R Notebooks that are tied to a specific release in the [`AlexsLemonade/training-modules`](https://github.com/AlexsLemonade/training-modules). You can use the `{{site.release_tag}}` convention used throughout the template.
+  - Use relative links to display rendered versions of R Notebooks in `completed-notebooks`.
```

Comment:
Probably for the other PR, but instructions for getting the rendered notebooks into this repo?

Reply:
Yeah, I plan to circle back to all of this as part of #112, but can definitely get it started in the other GHA PR if you prefer